addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -88,6 +91,20 @@ def _are_you_inside(self):
         for channel in self:
             channel.are_you_inside = self.env.user in channel.user_ids
 
+    @api.depends("channel_ids.livechat_end_dt")
+    def _compute_livechat_ongoing_sessions_count(self):
+        count_by_channel = self.env["discuss.channel"]._read_group(
```

Comment:
Note: summing "user_ids.livechat_ongoing_session_count" is not enough because agents leave the channel when they are overloaded.